### PR TITLE
Import Redcar and Cleveland (closes #2144)

### LINKED ISSUE #2144: Import E06000003-Redcar and Cleveland Borough Council
Xpress DC

### PR DESCRIPTION
**Pull request by alexsdutton**

### CODE REVIEW

**Comment by alexsdutton** [comment]
Argh. Ignore this for now.

**Comment by alexsdutton** [comment]
Good for review now :-)

File: polling_stations/apps/data_collection/management/commands/import_redcar_cleveland.py [hunk 1]
```diff
@@ -3,27 +3,55 @@
...
+            "10023901711",  # TS116BF -> TS68DN : 2 Rosedene Mews, Redcar Road, Marske by the Sea, Redcar
+            "10023903438",  # TS121AT -> TS121AU : Windsor Hall, Windsor Road, Saltburn by the Sea
+        ]:
+            rec["accept_suggestion"] = False
 
-        if uprn in ["10023906550"]:
+        if uprn in [
+            "10034529574",
+            "200002524170",
+        ]:
```
Comment (GeoWill) [inline polling_stations/apps/data_collection/management/commands/import_redcar_cleveland.py]: These can be rolled into the list above.

File: polling_stations/apps/data_collection/management/commands/import_redcar_cleveland.py [hunk 2]
```diff
@@ -3,27 +3,55 @@
...
+            "10023902773",  # TS122EA -> TS122BG : 2 Oak Mews, The Hills, Skelton-in-Cleveland, Saltburn by the Sea
+            "200002523768",  # TS122HD -> TS122HE : The Mill, Marske Lane, Skelton-in-Cleveland, Saltburn by the Sea
+            "200002523076",  # TS122BG -> TS122BH : Rosemont, Swilly Lane, Skelton-in-Cleveland, Saltburn by the Sea
+            "100110776824",  # TS121JG -> TS121HA : Toll Bridge Cottage, Saltburn Lane, Saltburn by the Sea
+            "100110676493",  # TS122JT -> TS121HA : White House Lodge, Saltburn Lane, Skelton-in-Cleveland
         ]:
             rec["accept_suggestion"] = True
+        #
+        if uprn in [
+            # "10023906550",
```
Comment (GeoWill) [inline polling_stations/apps/data_collection/management/commands/import_redcar_cleveland.py]: This should be un-commented or deleted.

**Comment by alexsdutton** [comment]
@GeoWill Ready for re-review :-)

**Pull request merged**